build/images/scripts/migrate_cni

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+CRICTL="crictl --runtime-endpoint unix:///run/containerd/containerd.sock"
+
+for file in /etc/cni/net.d/*; do
+    if [ "$file" != "/etc/cni/net.d/10-antrea.conflist" ]; then
+        mv "$file" "$file".bak
+    fi
+done
+
+# Remove rules of other CNI in CHAIN CNI-HOSTPORT-DNAT.
```

Comment:
of other CNI -> added by the previous CNI

Reply:
Done.